bus/bus.go

```diff
@@ -585,6 +586,22 @@ func (b *bus) setRedundancySettings(rs RedundancySettings) error {
 	}
 }
 
+func (b *bus) contractsAncestorsHandlerGet(jc jape.Context) {
```

Comment:
GET
(actually I don't add the HTTP verb unless there's multiple verbs for the same route, so in this case I'd drop the `Get` entirely)